dist/index.js

```diff
@@ -28704,15 +28700,21 @@ let client = undefined;
 
 async function getLicensingClient() {
     core.debug('Getting Licensing Client...');
+    let useLicenseClient = false;
     const editorPath = process.env.UNITY_EDITOR_PATH;
-    const version = process.env.UNITY_EDITOR_VERSION || editorPath.match(/(\d+\.\d+\.\d+[a-z]?\d?)/)[0];
-    core.debug(`Unity Editor Path: ${editorPath}`);
-    core.debug(`Unity Version: ${version}`);
-    await fs.access(editorPath, fs.constants.X_OK);
+    if (editorPath) {
+        core.debug(`Unity Editor Path: ${editorPath}`);
+        await fs.access(editorPath, fs.constants.X_OK);
+        const version = process.env.UNITY_EDITOR_VERSION || editorPath.match(/(\d+\.\d+\.\d+[a-z]?\d?)/)[0];
```

Comment:
I think it's good practice to leave a comment example of what the regex is trying to parse.

// Example: 2020.3.13f1

Reply:
regex is parsing version. hence the name of the variable it sets.